### DISCUSSION TITLE
Custom surround shortcuts?

### DISCUSSION THREAD

**Discussion by afedorychak** [JetBrains/ideavim#505] [Q&A]
Is there any way to invoke the surround plugin with a given pre-defined character using a custom key sequence?

E. g., if I want to type `gw` and make it work as if I typed `ysiw"` ?

This binding works in VIM, but doesn't work in IdeaVim: `nmap gw ysiw"`.

**Comment by AlexPl292** [comment]
Have you tries to use a regular vim mappings via `map`?

**Reply by afedorychak** [reply]
Yes, updated my question.

**Reply by AlexPl292** [reply]
This will be fixed in a next release https://youtrack.jetbrains.com/issue/VIM-2671/Cant-use-surround-plugin-in-mappings
